inference/habana/README.md

```diff
@@ -0,0 +1,26 @@
+# Inference with Intel Habana Gaudi
```

Comment:
can you please update the existing setup.md and serve.md in /docs instead of creating this new document?